src/main/java/hw8/entities/MetalsColorsInput.java

```diff
@@ -0,0 +1,81 @@
+package hw8.entities;
+
+import com.epam.jdi.tools.DataClass;
+import hw8.Utils;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.lang.String.format;
+
+public class MetalsColorsInput extends DataClass<MetalsColorsInput> {
+    private List<String> summary;
+    private List<String> elements;
+    private String color;
+    private String metals;
+    private List<String> vegetables;
+
+    private static Map<String, Integer> summaryValuesIndexes = new HashMap<>();
```

Comment:
What is the reason for using this Map?

Reply:
I've tried to use JDI functionality.
Unfotunately, radio buttons on the Metals & Colors page are not "standard": they don't contain their values inside the tag, only as a text. So the only way to select them with JDI method is to use index (RadioButtons.select(int index)). 
But in the Input file contains values, not indexes.
Also the radio buttons on the page are divided into 2 groups ("odds" and "even"), so their indexes repeat.
That's why I introduced Map to establish the relationship between value and its index in the group.

Reply:
Deleted.
SummuryEntity is not used anymore